frontend/components/auction/surplus/SurplusAuctionsTable.vue

```diff
@@ -10,6 +10,8 @@
             :custom-row="customRowEvents"
             :get-popup-container="() => $el"
             :locale="{ emptyText: 'No active auctions' }"
+            :table-layout="auto"
+            :scroll="{ x: 'max-content' }"
```

Comment:
What is `max-content` string? According to the docs, `scroll` keys only accept numbers or a boolean:
<img width="920" alt="Screenshot 2022-11-28 at 09 29 32" src="https://user-images.githubusercontent.com/3393626/204230344-adb482e1-f50f-4b08-8a12-5e4d50225e10.png">
https://2x.antdv.com/components/table#API